src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ToggleSwitchControl/ToggleSwitch_TemplateReuse.xaml.cs

```diff
@@ -0,0 +1,38 @@
+﻿using SamplesApp.Windows_UI_Xaml_Controls.ToggleSwitchControl.Models;
+using Uno.UI.Samples.Controls;
+using Uno.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace UITests.Shared.Windows_UI_Xaml_Controls.ToggleSwitchControl
+{
+	[SampleControlInfo("ToggleSwitch", "ToggleSwitch_TemplateReuse", typeof(ToggleSwitchViewModel))]
+	public sealed partial class ToggleSwitch_TemplateReuse : Page
+	{
+		public ToggleSwitch_TemplateReuse()
+		{
+			this.InitializeComponent();
+
+			FeatureConfiguration.Page.IsPoolingEnabled = true;
+			FrameworkTemplatePool.IsPoolingEnabled = true;
```

Comment:
This feature needs to be reset when the control is unloaded (the sample app disables it by default). Best it to do this on Loaded and Unloaded events.